PictureApp/Controller/OptionsViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  OptionsViewController.swift
+//  PictureApp
+//
+//  Created by Irina Gorbachenko on 26.11.2021.
+//
+
+import UIKit
+import NetworkLayer
+
+protocol OptionsViewControllerDelegate : AnyObject {
+    func optionsViewController (_ controller: OptionsViewController, didSelectOrderStrategy order: Order)
+    func didFilterSelected()
+    
+}
+ 
+enum Order :String{
+
+  case popular = "popular"
+  case latest = "latest"
+
+}
+
+class OptionsViewController: UIViewController {
+    
+    @IBOutlet var sortButton: UIButton!
+    var isTapped :Bool = true
```

Comment:
I think you could replace it with the latest chosen sort.
Because we have a default value .popular

Reply:
Done